src/Symfony/Component/Security/Http/Firewall/RememberMeListener.php

```diff
@@ -73,6 +77,12 @@ public function handle(GetResponseEvent $event)
 
         try {
             $token = $this->authenticationManager->authenticate($token);
+            if ($this->sessionStrategy
+                && $request->hasSession()
+                && $request->getSession()->isStarted()
+            ) {
```

Comment:
should be `null !== $this->sessionStrategy` and please keep the `if` on one line